pkg/processtree/processtree.go

```diff
@@ -76,13 +80,6 @@ func (p Process) Exited() bool {
 	return p.ExitTime > 0
 }
 
-type ProcessTreeCollector interface {
-	ProcessStarted(eventTime time.Time, containerID string, p Process)
```

Comment:
By removing all of this methods, please also get rid of the gorouting started for exporting process tree events. The init method of the process tree runs in sync, so no need for parallelism.